WooCommerce/Classes/ViewRelated/Dashboard/Stats v4/StoreStatsV4PeriodViewController.swift

```diff
@@ -354,6 +359,30 @@ private extension StoreStatsV4PeriodViewController {
         // Data
         updateStatsDataToDefaultStyles()
 
+        // Taps
+        if timeRange.isCustomTimeRange {
```

Comment:
We should also check if the site is JCP or doesn't have Jetpack connection and not show the alert for those cases.

Reply:
Thanks, I updated the logic in [dd8989c](https://github.com/woocommerce/woocommerce-ios/pull/12226/commits/dd8989ccd0177e943a3554acc147ef88d7657afa) to handle these two things:

1. Make sure non-Jetpack and Jetpack CP sites do not show the info alert when tapped,
2. Make sure info alert is also not shown when in custom range wpcom/Jetpack sites, but individual graph value is selected.

I updated the test instructions above to include these.